src/licensedcode/data/licenses/bsd-3-clause-flex.LICENSE

```diff
@@ -0,0 +1,134 @@
+---
+key: bsd-3-clause-flex
+short_name: BSD 3-Clause Flex variant
+category: Permissive
+owner: Regents of the University of California
+name: BSD 3-Clause Flex variant
+notes: |
```

Comment:
See https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/licenses/flex-2.5.LICENSE which has a slightly different license text, but has rules which are matching.

Reply:
@AyanSinhaMahapatra   Rather than create a new scancode license, I think that we should update the existing scancode flex-2.5 license to match this new SPDX license: 
1. replace the license text in the scancode licensedb
2. update the spdx_license_key to BSD-3-Clause-flex to replace the current LicenseRef

I am aware that in the license text we normally avoid copyrights and references to specific organizations and code projects, but I think that is appropriate for this license, which we would call a component_license.